scout/parse/variant/genotype.py

```diff
@@ -118,9 +118,9 @@ def parse_genotype(variant, ind, pos):
         try:
             alt_value = int(values[1])
             ref_value = int(values[0])
-            if not alt_value < 0:
+            if alt_value >= 0:
```

Comment:
When is the value less than 0 reads?

Reply:
Well there is this slightly weird missing value -1, which I suppose is what is sought here, otherwise it would be a weird beast.